README.md

```diff
@@ -53,6 +53,18 @@ cargo run [-- CONFIG.json]
 
 for config see [this](./ricochet-headarches.json) and [that](./ricochet.json) as example
 
+# set logs at runtime
+
+`"scope"` is an array of what you want to debugs, likes `["nets", "vms"]`
+`"dir"` is use to set if you want to see input arguments, or output, must contain eithe `"in", "out" or "all"`
```

Comment:
`"dir"` is used to set if you want to see input or output arguments, either `"in"`, `"out"` or `"all"`